jdbc-perf-logger-driver/src/main/java/ch/sla/jdbcperflogger/logger/PerfLoggerRemoting.java

```diff
@@ -31,15 +35,25 @@
 
     final static Set<LogSender> senders = new CopyOnWriteArraySet<LogSender>();
     final static Map<LoggingConnectionInvocationHandler, ConnectionInfo> connectionToInfo = new WeakHashMap<LoggingConnectionInvocationHandler, ConnectionInfo>();
+    final static List<Closeable> createdThreads = new ArrayList<Closeable>();
 
-    static {
+    public static void start() {
         final Integer serverPort = DriverConfig.INSTANCE.getServerPort();
         if (serverPort != null) {
-            PerfLoggerServerThread.spawn(serverPort);
+            createdThreads.add(PerfLoggerServerThread.spawn(serverPort));
         }
-
         for (final InetSocketAddress clientAddress : DriverConfig.INSTANCE.getClientAddresses()) {
-            PerfLoggerClientThread.spawn(clientAddress);
+            createdThreads.add(PerfLoggerClientThread.spawn(clientAddress));
+        }
+    }
+
+    public static void stop() {
+        for (final Closeable thread : createdThreads) {
+            try {
+                thread.close();
```

Comment:
the thread should be removed from createdThreads after closing it (or clear createdThreads entirely) to avoid a memory leak after several start/stops...